examples/images_contours_and_fields/pcolormesh_levels.py

```diff
@@ -54,8 +115,6 @@
 # don't overlap
 fig.tight_layout()
```

Comment:
Call the initial figure with `constrained_layout` instead?

Reply:
I was told to not change *every* example that used `tight_layout` :wink: